mscore/resourceManager.h

```diff
@@ -46,5 +46,27 @@ private slots:
     void uninstallExtension();
    };
 
+class ExtensionFileSize : public QTableWidgetItem
+   {
+      int _int;
```

Comment:
Maybe it would be better to call this variable `_size` or something like this, `_int` is not very informative name for it.